osdpctl/src/config.rs

```diff
@@ -255,6 +259,8 @@ impl PdConfig {
     }
 }
 
+/// Configuration for the
```

Comment:
Some words missing here?

Reply:
Sorry, overlooked that one.

Did not notice that the other things in this crate also don't have doc-comments (which is reasonable for a binary)

```suggestion
```